### DISCUSSION TITLE
✅ How to add a `Tag` to an API Route

### DISCUSSION THREAD

**Discussion by TheWisestOne** [RobinTail/express-zod-api#564] [Ideas]
Hi, I'm enjoying digging into this library, but am struggling with one detail. I can't seem to figure out how to set the `tags` for an API endpoint when using `endpointFactory.build`.
by `tags`, I'm talking about OpenApi tags: https://swagger.io/docs/specification/grouping-operations-with-tags/
This helps group the API endpoints in the generated Swagger UI. Is this feature available in this library?

**Comment by RobinTail** [comment]
@TheWisestOne ,

tags are not yet supported by the OpenAPI generator and `EndpointsFactory` methods.
Not sure yet what might be the best way to implement this feature preserving the constraints across all the endpoints.
I'm open to suggestions.

**Comment by RobinTail** [comment]
Dear @TheWisestOne,
the feature you asked for has been released in v8.1.0 🚀
